https://issues.redhat.com/browse/ACM-24717 Add managed ns doc (#8341)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/set_managed_ns.adoc

[#set-managed-ns]
= Setting managed namespaces for _ManagedClusterSets_

> You can define on or more managed namespaces in a `ManagedClusterSet`. If you specify managed namespaces, they are automatically created on every managed cluster that belongs to the set.

Reviewer: Are there any prereqs? Reminder to add the required access for this task
```suggestion
You can define one or more managed namespaces in a `ManagedClusterSet`. If you specify managed namespaces, they are automatically created on every managed cluster that belongs to the set.
```

> See the following example of a `ManagedClusterSet` with managed namespaces:

Reviewer: ```suggestion
Complete the following steps to create namespaces for your `ManagedClusterSet` resource:

. Add the name of your managed namespaces. See the following example of a `ManagedClusterSet` resource where the `my-cluster-ns1` and `my-cluster-ns2` managed namespaces are defined:
```

[source,yaml]
----
apiVersion: cluster.open-cluster-management.io/v1beta2
kind: ManagedClusterSet
metadata:
  name: my-clusterset
spec:
  clusterSelector:
    selectorType: ExclusiveClusterSetLabel
  managedNamespaces:
  - name: my-clusterset-ns1
  - name: my-clusterset-ns2
----

> You can check the managed namespaces in the `status` of each `ManagedCluster` that is part of the set.

Reviewer: I would also combine lines 22 and 24 
```suggestion
. Check the managed namespaces in the `status` of each `ManagedCluster` that is part of the set.
```

See the following `ManagedCluster` `status` with managed namespaces. The `conditions.type` shows if the namespace is available on the managed cluster:

[source,yaml]
----
apiVersion: cluster.open-cluster-management.io/v1
kind: ManagedCluster
metadata:
  labels:
    cluster.open-cluster-management.io/clusterset: my-clusterset
  name: cluster1
status:
  managedNamespaces:
  - clusterSet: my-clusterset
    conditions:
    - lastTransitionTime: "2025-09-26T03:15:52Z"
      message: Namespace successfully applied and managed
      reason: NamespaceApplied
      status: "True"
      type: NamespaceAvailable
    name: my-clusterset-ns1
  - clusterSet: my-clusterset
    conditions:
    - lastTransitionTime: "2025-09-26T03:15:52Z"
      message: Namespace successfully applied and managed
      reason: NamespaceApplied
      status: "True"
      type: NamespaceAvailable
    name: my-clusterset-ns2
----

*Important:* Namespaces that you create on managed clusters are not deleted, even in the followiwing situations:

- You remove the namespace from the `ManagedClusterSet` specification.
- You move the managed cluster to a different `ManagedClusterSet`.

> - You detach the managed cluster from the hub cluster.

Reviewer: I recommend that this move this closer to the beginning of the doc. Make it a part of the description. So before the prereqs. Let me know your thoughts

Author: That's a good idea, I'll move it

[#set-managed-ns]
= Setting managed namespaces for _ManagedClusterSets_

> You can define one or more managed namespaces in a `ManagedClusterSet`. If you specify managed namespaces, they are automatically created on every managed cluster that belongs to the set.

Reviewer 2: seems like and extra space here.

This description is missing the reason/why the user wants to do this. You can almost always find this in the Epic or Feature, if not, the engineers can explain it.

Author: Updated
